cluster/gce/configure-vm.sh

```diff
@@ -949,7 +949,7 @@ EOF
   if [[ ! -z "${KUBELET_APISERVER:-}" ]] && [[ ! -z "${KUBELET_CERT:-}" ]] && [[ ! -z "${KUBELET_KEY:-}" ]]; then
     cat <<EOF >>/etc/salt/minion.d/grains.conf
   kubelet_api_servers: '${KUBELET_APISERVER}'
-  cbr-cidr: 10.123.45.0/30
+  cbr-cidr: 10.123.45.0/29
```

Comment:
I also think that 1.3 was the first release where we registered the master into the cluster. So 1.2 --> 1.3 upgrades went from the master being standalone to the master being registered. And 1.3 --> 1.4 is the first upgrade where the master was registered before and after the upgrades. It's possible that there is a bug with the master node registration during upgrades that is causing the CIDR not to be set properly by the kubelet.